snapshotter/devmapper/devmapper.go

```diff
@@ -200,10 +201,6 @@ func (dm *Snapshotter) Close() error {
 		result = multierror.Append(result, err)
 	}
 
-	if err := dm.pool.Close(context.Background(), false, false); err != nil {
```

Comment:
No equivalent is necessary?

Reply:
No, we want pool remain exist after we shutdown snapshotter (so we don't loose data)

Reply:
Got it, was this an error in the previous implementation then?

Reply:
It actually did nothing because both `removeDevices` and  `removePool` were set to false.